src/shared/db.py

```diff
@@ -204,29 +246,51 @@ def __init__(self, table_name: str, region: str, host: Optional[str] = None):
     def new_user(
         self, uid: str, origin_system: str, cust_id: Optional[str] = None
     ) -> SubHubDeletedAccountModel:
-        return self.model(
+        logger.debug(
+            f"Enter {SubHubDeletedAccount.__class__.__name__}::{SubHubAccount.new_user.__name__} ({uid}, {origin_system}, {cust_id})"
+        )
+        user = self.model(
             user_id=uid,
             cust_id=cust_id,
             origin_system=origin_system,
             customer_status="deleted",
         )
+        logger.debug(
+            f"Exit {SubHubDeletedAccount.__class__.__name__}::{SubHubAccount.new_user.__name__} ({uid}, {origin_system}, {cust_id})",
+            user=user,
+        )
+        return user
 
     def get_user(self, uid: str) -> Optional[SubHubDeletedAccountModel]:
+        logger.debug(
+            f"Enter {SubHubDeletedAccount.__class__.__name__}::{SubHubAccount.get_user.__name__} ({uid})"
+        )
+        subscription_user: Optional[Any] = None
         try:
-            subscription_user: Optional[Any] = self.model.get(uid, consistent_read=True)
-            return subscription_user
+            subscription_user = self.model.get(uid, consistent_read=True)
         except DoesNotExist:
             logger.error("get user", uid=uid)
-            return None
+        logger.debug(
+            f"Enter {SubHubDeletedAccount.__class__.__name__}::{SubHubAccount.get_user.__name__} ({uid})"
```

Comment:
is `SubHubAccount.get_user.__name__` == `subscription_user`?  If so, why not just log that after you get it?